src/mumble/PulseAudio.h

```diff
@@ -21,15 +32,96 @@ struct PulseAttenuation {
+									   void *userdata);
+	pa_context_state_t (*context_get_state)(const pa_context *c);
+	pa_operation *(*context_get_server_info)(pa_context *c, pa_server_info_cb_t cb, void *userdata);
+	pa_operation *(*context_get_sink_info_by_name)(pa_context *c, const char *name, pa_sink_info_cb_t cb,
+												   void *userdata);
+	pa_operation *(*context_get_sink_info_list)(pa_context *c, pa_sink_info_cb_t cb, void *userdata);
+	pa_operation *(*context_get_sink_input_info_list)(pa_context *c, pa_sink_input_info_cb_t cb, void *userdata);
+	pa_operation *(*context_get_source_info_list)(pa_context *c, pa_source_info_cb_t cb, void *userdata);
+	pa_operation *(*context_set_sink_input_volume)(pa_context *c, uint32_t idx, const pa_cvolume *volume,
+												   pa_context_success_cb_t cb, void *userdata);
+	void (*context_set_state_callback)(pa_context *c, pa_context_notify_cb_t cb, void *userdata);
+	void (*context_set_subscribe_callback)(pa_context *c, pa_context_subscribe_cb_t cb, void *userdata);
+
+	pa_stream *(*stream_new)(pa_context *c, const char *name, const pa_sample_spec *ss, const pa_channel_map *map);
+	void (*stream_unref)(pa_stream *s);
+	int (*stream_connect_playback)(pa_stream *s, const char *dev, const pa_buffer_attr *attr, pa_stream_flags_t flags,
+								   const pa_cvolume *volume, pa_stream *sync_stream);
+	int (*stream_connect_record)(pa_stream *s, const char *dev, const pa_buffer_attr *attr, pa_stream_flags_t flags);
+	int (*stream_disconnect)(pa_stream *s);
+	int (*stream_peek)(pa_stream *p, const void **data, size_t *nbytes);
+	int (*stream_write)(pa_stream *p, const void *data, size_t nbytes, pa_free_cb_t free_cb, int64_t offset,
+						pa_seek_mode_t seek);
+	int (*stream_drop)(pa_stream *p);
+	pa_operation *(*stream_cork)(pa_stream *s, int b, pa_stream_success_cb_t cb, void *userdata);
+	pa_stream_state_t (*stream_get_state)(const pa_stream *p);
+	pa_context *(*stream_get_context)(const pa_stream *p);
+	const pa_sample_spec *(*stream_get_sample_spec)(pa_stream *s);
+	const pa_channel_map *(*stream_get_channel_map)(pa_stream *s);
+	const pa_buffer_attr *(*stream_get_buffer_attr)(pa_stream *s);
+	void (*stream_set_state_callback)(pa_stream *s, pa_stream_notify_cb_t cb, void *userdata);
+	void (*stream_set_read_callback)(pa_stream *p, pa_stream_request_cb_t cb, void *userdata);
+	void (*stream_set_write_callback)(pa_stream *p, pa_stream_request_cb_t cb, void *userdata);
+
+	pa_operation *(*ext_stream_restore_read)(pa_context *c, pa_ext_stream_restore_read_cb_t cb, void *userdata);
+	pa_operation *(*ext_stream_restore_write)(pa_context *c, pa_update_mode_t mode,
+											  const pa_ext_stream_restore_info data[], unsigned n,
+											  int apply_immediately, pa_context_success_cb_t cb, void *userdata);
+
+public:
+	PulseAudio();
+};
 
 class PulseAudioSystem : public QObject {
 private:
 	Q_OBJECT
 	Q_DISABLE_COPY(PulseAudioSystem)
 protected:
 	void wakeup();
+
+	PulseAudio pa;
```

Comment:
I would prefer a longer variable name that is more descriptive on its own :thinking: 
E.g. `pulseAudio`

Additionally using a `m_` prefix to mark it as a member variable might be a good thing as well :point_up: